core/ui/src/main/java/com/yapp/core/ui/util/DateTime.kt

```
@@ -0,0 +1,60 @@
+                hour12
+            )
+            30 -> context.getString(
+                if (isAm) R.string.time_format_half_hour else R.string.time_format_half_hour_pm,
+                hour12
+            )
+            else -> context.getString(
+                if (isAm) R.string.time_format_hour_minute else R.string.time_format_hour_minute_pm,
+                hour12, minute
+            )
+        }
+    } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)